.github/workflows/makefile.yml

```diff
@@ -63,7 +63,22 @@ jobs:
     # When building for coverage, explicitly tell Clang to use the right include
     # path, because Clang's heuristics often fail to deduce it automatically.
     - name: Build coverage report
+      id: build-coverage
       env:
         CPLUS_INCLUDE_PATH: /usr/i686-linux-gnu/include/c++/${{ env.COVERAGE_LIBSTDCPP_VERSION }}/i686-linux-gnu/
-      run:
-        ./coverage-report.sh
+      run: |
+        ./coverage-report.sh | tee coverage-output.txt
+        echo "::set-output name=LINE_COVERAGE_PERCENT::$(
+          scripts/parse-coverage-output.py < coverage-output.txt
+        )"
+
+    # Post an autogenerated comment on the Pull Request (in case the action is
+    # run because of one).
+    - name: Comment on pull request
+      if: github.event.pull_request
+      uses: peter-evans/create-or-update-comment@v2
+      with:
+        issue-number: ${{ github.event.pull_request.number }}
+        body: |
+          __Continuous Integration__: All tests passed.
```

Comment:
Just curious - what will happen if coverage will fail or build fails? This message will not pop, right?

Reply:
Yeah indeed, but when the action fails GitHub sends an email about the failed workflow.
When everything passes, there's no email, hence a separate comment seems useful; also we're posting the most important excerpt of the otherwise huge logs.